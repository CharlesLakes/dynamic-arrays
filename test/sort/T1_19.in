10
74 74 69 76 67 42 99 32 82 85