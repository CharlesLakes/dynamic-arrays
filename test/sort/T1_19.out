32 42 67 69 74 74 76 82 85 99